pytorch_lightning/loops/fit_loop.py

```diff
@@ -274,3 +282,11 @@ def _check_checkpoint_callback(self, should_update: bool, is_last: bool = False)
 
             for cb in callbacks:
                 cb.on_validation_end(self.trainer, model)
+
+    def state_dict(self) -> Dict:
+        if not self.is_connected:
+            raise MisconfigurationException("The Trainer should be connected to loop to retrieve the state_dict.")
+        return {self.epoch_loop.name: self.epoch_loop.state_dict()}
```

Comment:
What is the benefit of letting each loop define its saved name when the parent gets the state dict?

Reply:
It thought it could be useful for tracking dynamically the names.

Reply:
Removed !